src/projetos/SugestaoService.ts

```
@@ -0,0 +1,32 @@
+  }
+
+  async findAll(): Promise<Sugestao[]> {
+    return this.get<Sugestao[]>()
+  }
+
+  async remove(id: number): Promise<void> {
+    return this.delete(`/${id}`)
+  }
+
+  async findMine(): Promise<Sugestao[]> {
+     return this.get<Sugestao[]>("/minhas-sugestoes")
```

ESLint

[error] Replace `·return·this.get<Sugestao[]>("/minhas-sugestoes"` with `return·this.get<Sugestao[]>('/minhas-sugestoes'`

(prettier/prettier)